File: frontend/src/types/stage.ts
export interface StageData {
  resultCode: number;
  resultMsg: string;
  result: StageResult[];
}

interface StageResult {
  stage: Stage;
  starCount: number;
}

interface Stage {
  stageId: number;
  stageName: string;
  stageImage: string;
  characterImage: string;
  description: string;
  targetStarCount: number;
}

File: backend/src/main/java/com/ssafy/chorongddara/db/entity/Stage.java
package com.ssafy.chorongddara.db.entity;

import lombok.*;

import javax.persistence.*;

@Entity
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Getter
public class Stage {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer stageId;
    @Column(nullable = false)
    private String stageName;
    @Column(nullable = false)
    private String stageImage;
    @Column(nullable = false)
    private String characterImage;
    private String description;
    private Integer targetStarCount;
}
